.github/workflows/test.yml

```diff
@@ -22,7 +22,9 @@ jobs:
       run: go build -v ./...
 
     - name: Test & Coverage
-      run: go test -race -coverprofile=coverage.txt -covermode=atomic -v ./...
+      run: |
+        go install github.com/ory/go-acc@v0.2.6
+        go-acc -o coverage.txt ./... -- -v --race
```

Comment:
Interesting, I didn't know about this tool.

Reply:
You can read about it here: https://www.ory.sh/golang-go-code-coverage-accurate/